src/components/themed/SideMenu.tsx

```diff
@@ -484,14 +483,3 @@ const OuterView = styled(View)<{ insets: { top: number; bottom: number } }>(() =
   paddingTop: props.insets.top,
   paddingBottom: props.insets.bottom
 }))
-
-export function SideMenu(props: DrawerContentComponentProps) {
-  const { navigation } = props
-
-  const { loggedIn } = useSelector(state => state.core.account)
-  React.useEffect(() => {
-    if (!loggedIn) navigation.navigate('login')
-  }, [loggedIn, navigation])
-
-  return loggedIn ? <SideMenuComponent {...props} /> : <LoadingSplashScreen />
-}
```

Comment:
I don't understand the motivation for leaving the SideMenu mounted while on the login screen?

Reply:
When the bug happens, the user can get stuck inside the app. All the scenes get replaced by spinners, and the side menu also gets replaced by a spinner. So, there is no button the user can press to escape this situation, aside from killing the whole app.

By letting the side menu be the side menu, even if the account is in a logged-out state, the user can press the "logout" button and escape.